packages/core/examples/inputGroupExample.tsx

```diff
@@ -86,14 +82,16 @@ export class InputGroupExample extends BaseExample<IInputGroupExampleState> {
             <div>
                 <div>
                     <InputGroup
+                        className={className}
                         disabled={disabled}
                         leftIconName="filter"
                         onChange={this.handleFilterChange}
                         placeholder="Filter histogram..."
-                        rightElement={clearButton}
+                        rightElement={filterValue ? <Spinner className={Classes.SMALL} /> : undefined}
                         value={filterValue}
                     />
                     <InputGroup
+                        className={className}
                         disabled={disabled}
                         placeholder="Enter your password..."
```

Comment:
it really does seem that input placeholders __do not support ellipsis__: https://css-tricks.com/almanac/selectors/p/placeholder/#article-header-id-4. they support a very minimal set of styling properties, and `text-overflow` is not among them.